Department.php

```diff
@@ -0,0 +1,30 @@
+<?php
+    class Department
+    {
+        public function __construct(private string $name, private $Employees)
+        {
+        }
+        public function get_name(): string{
+            return $this->name;
+        }
+        public function get_count_employee(): int
+        {
+            return count($this->Employees);
+        }
+        public function total_salary(): int
+        {
+            $total_salary = 0;
```

Comment:
Переменные в коде тоже должны быть в camelCase
```suggestion
            $totalSalary = 0;
```